src/app/features/test-records/views/test-record/test-record.component.html

```diff
@@ -17,3 +17,5 @@
     <app-test-amendment-history [testRecord]="testResult"></app-test-amendment-history>
   </section>
 </section>
+
+<router-outlet></router-outlet>
```

Comment:
For future reference, we don't need the router outlet if we aren't using test record section templates (see routing module)